tests/unit/Validator/ManagerTest.php

```diff
@@ -21,12 +24,18 @@ class ManagerTest extends UnitTest
         testItIsAManager as traitTestItIsAManager;
     }
 
+    private $application;
+
     /**
      * @return void
      */
     public function _before(): void
     {
+        /** @var Manager $manager */
         $this->manager = new Manager();
+
+        /** @var Application $application */
+        $this->application = Application::init($this->tester->getConfig());
```

Comment:
Application is already initialized by the SDK codeception Module, it's not necessary to initiate it again. Use the helper methods to get the application, service manager, etc.